discord/appinfo.py

```diff
@@ -122,14 +131,31 @@ class AppInfo:
         'privacy_policy_url',
     )
 
-    def __init__(self, state, data):
+    if TYPE_CHECKING:
+        id: int
+        name: str
+        owner: User
+        team: Optional[Team]
+        description: Optional[str]
```

Comment:
Discord API docs specify this as string explicitly? should this still be optional?